astropy/utils/iers/iers.py

```diff
@@ -351,8 +371,18 @@ def _check_interpolate_indices(self, indices_orig, indices_clipped, max_input_mj
         class because it has different requirements.
         """
         if np.any(indices_orig != indices_clipped):
-            raise IERSRangeError('(some) times are outside of range covered '
-                                 'by IERS table.')
+            if conf.iers_degraded_accuracy == 'error':
+                msg = ('(some) times are outside of range covered by IERS table. Cannot convert '
+                       'with full accuracy. To allow conversion with degraded accuracy '
+                       'set astropy.utils.iers.conf.iers_degraded_accuracy '
+                       ' to "warn" or "silent"')
```

Comment:
A truly embarrassing nitpick, but since you will do one more commit anyways... 😅 
```suggestion
                       'to "warn" or "silent"')
```

Reply:
It also may be worth adding something like "For more information about setting this configuration parameter or controlling its value globally, see the Astropy configuration system documentation https://docs.astropy.org/en/stable/config/index.html" for more context?

Reply:
Done